### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by DuttonMa**
Amended instructions for setup of initial dev environment.
Added instructions for setting up migrations.
I need to add files to the migrate instructions but that will follow shortly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +16,44 @@ We recommend Lando for local development. To get started, ensure you have the fo
...
+`platform db:dump` (you'll need to select project, environment and required site e.g. 'uregni')
+
+`lando db-import -h <site name> <downloaded db>` (where 'site name' may be 'uregni', 'liofa' etc)
+
+## Running migrations
+
+You will first need to get hold of a Drupal 7 database dump for your chosen site to act as the source of the migration.
+We will take Uregni as an example and assume that we have a dump file 'uregni.sql'.
+
+1. Import the database into the 'uregni7' database host:
```
Comment (omahm) [inline README.md:35]: I can see people skimming over this and importing over the D8 DB. I'd maybe stress this is importing to the D7 database container.

File: README.md [hunk 2]
```diff
@@ -16,14 +16,44 @@ We recommend Lando for local development. To get started, ensure you have the fo
...
+`lando db-import -h uregni7 uregni.sql`
+
+2. Install the migrate_upgrade module
+`lando drupal moi migrate_upgrade` (or you could do this from the admin screens)
+
+3. Make sure that you are in the appropriate site directory e.g. web/sites/uregni and run this command:
+`lando drush migrate-upgrade --legacy-db-url=mysql://drupal7:drupal7@uregni7/drupal7 --legacy-root=http://www.uregni.goc.uk --configure-only`
```
Comment (soda736) [inline README.md:42]: Should the URL should be uregni.gov.uk.

File: README.md (hunk 1 above)
Comment (DuttonMa) [reply README.md]: Good point, have changed that now

File: README.md (hunk 2 above)
Comment (DuttonMa) [reply README.md]: Ouch, good spot !
Thanks, have fixed that now.

**Pull request merged**